CHANGELOG.md

```diff
@@ -4,10 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [3.0.2]
 ### Removed
 - “Sync to remote environments” checkboxes.
 - "Delivery Event Listeners"
+- Configuration of delivery properties in DeliveryFactory
+### Added
+- A new event handler to create backup of QTI test package when delivery is created
+- A new property of Remote Environment - `Publishing Enabled`
+- A new UI interface to publish delivery to selected remote environment(s) when it's ready
```

Comment:
Not sure if `when it's ready` should be specified, since if there's a delivery then by design it's ready to be published

Reply:
Fixed